src/catalog/postgres/builder.cpp

```diff
@@ -294,6 +308,33 @@ Schema Builder::GetTypeTableSchema() {
   return Schema(columns);
 }
 
+Schema Builder::GetLanguageTableSchema() {
+  std::vector<Schema::Column> columns;
+
+  columns.emplace_back("lanoid", type::TypeId::INTEGER, false, MakeNull(type::TypeId::INTEGER));
+  columns.back().SetOid(LANOID_COL_OID);
+
+  columns.emplace_back("lanname", type::TypeId::VARCHAR, MAX_NAME_LENGTH, false, MakeNull(type::TypeId::VARCHAR));
+  columns.back().SetOid(LANNAME_COL_OID);
+
+  columns.emplace_back("lanispl", type::TypeId::BOOLEAN, true, MakeNull(type::TypeId::BOOLEAN));
+  columns.back().SetOid(LANISPL_COL_OID);
+
+  columns.emplace_back("lanpltrusted", type::TypeId::BOOLEAN, true, MakeNull(type::TypeId::BOOLEAN));
+  columns.back().SetOid(LANPLTRUSTED_COL_OID);
```

Comment:
These should probably be not nullable.  The values for the bootstrapped languages should be known at compile time (i.e. false for `ispl` and true for `trusted`).